### DISCUSSION TITLE
Pro 会员 pdf 每月用量

### DISCUSSION THREAD

**Discussion by chengzyzy** [immersive-translate/immersive-translate#1697] [Help]
### 概述 | Summary

![image](https://github.com/immersive-translate/immersive-translate/assets/30223515/90e08100-1747-43a8-915b-2b8da5d506b9)
每月100页 是什么意思? 翻译的额度?
我有一个1500页很复杂的文档需要翻译 ,我可以用自己的openapi 再用 pdf pro的排版吗

**Comment by keyboard3** [comment]
100页 AI 解析PDF的额度。如果是论文pdf的话可以去尝试尝试

**Comment by theowenyoung** [comment]
pdf pro的页数是单独计费的，按页数计费，会员套餐内不够的话，需要额外购买页数。
